tests/model_fields/tests.py

```diff
@@ -904,6 +904,17 @@ def test_genericipaddressfield_formfield_protocol(self):
         with self.assertRaises(ValidationError):
             form_field.clean('127.0.0.1')
 
+    def test_mixed_ipv46(self):
```

Comment:
Can the tests go in `tests/utils_tests/test_ipv6.py`? (possibly some are redundant with what's already there)